specification/signalr/resource-manager/Microsoft.SignalRService/preview/2021-04-01-preview/signalr.json

```diff
@@ -0,0 +1,2224 @@
+        },
+        "size": {
+          "description": "Not used. Retained for future use.",
+          "type": "string",
+          "readOnly": true
+        },
+        "family": {
+          "description": "Not used. Retained for future use.",
+          "type": "string",
+          "readOnly": true
+        },
+        "capacity": {
+          "format": "int32",
+          "description": "Optional, integer. The unit count of the resource. 1 by default.\r\n\r\nIf present, following values are allowed:\r\n    Free: 1\r\n    Standard: 1,2,5,10,20,50,100",
+          "type": "integer"
+        }
+      }
+    },
+    "ServerlessUpstreamSettings": {
+      "description": "The settings for the Upstream when the service is in server-less mode.",
+      "type": "object",
+      "properties": {
+        "templates": {
+          "description": "Gets or sets the list of Upstream URL templates. Order matters, and the first matching template takes effects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/UpstreamTemplate"
+          }
+        }
+      }
+    },
+    "ServiceKind": {
+      "description": "The kind of the service - e.g. \"SignalR\" for \"Microsoft.SignalRService/SignalR\"",
+      "enum": [
+        "SignalR",
+        "RawWebSockets"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "ServiceKind",
+        "modelAsString": true
+      }
+    },
+    "ServiceSpecification": {
+      "description": "An object that describes a specification.",
+      "type": "object",
+      "properties": {
+        "metricSpecifications": {
+          "description": "Specifications of the Metrics for Azure Monitoring.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/MetricSpecification"
+          }
+        },
+        "logSpecifications": {
+          "description": "Specifications of the Logs for Azure Monitoring.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LogSpecification"
+          }
+        }
+      }
+    },
+    "ShareablePrivateLinkResourceProperties": {
+      "description": "Describes the properties of a resource type that has been onboarded to private link service",
+      "type": "object",
+      "properties": {
+        "description": {
+          "description": "The description of the resource type that has been onboarded to private link service",
+          "type": "string"
+        },
+        "groupId": {
+          "description": "The resource provider group id for the resource that has been onboarded to private link service",
+          "type": "string"
+        },
+        "type": {
+          "description": "The resource provider type for the resource that has been onboarded to private link service",
+          "type": "string"
+        }
+      }
+    },
+    "ShareablePrivateLinkResourceType": {
```

Comment:
>ShareablePrivateLinkResourceType [](start = 5, length = 32)

Just to confirm, is a shareable type an entity that can later be shared? is that the difference between shareable and shared definitions?

Reply:
The resource type itself is sharable, which is introduced by Network team. A shared private link can be used by different resource providers.